### PR TITLE
Fix #5 : Update readme

### LINKED ISSUE #5: Update README
- Installation instructions
- UI screenshot
- Endpoints
- License details

### PR DESCRIPTION
**Pull request by amrita019**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,81 @@
...
+</p>
+
 # nodecloud-web
+
+nodecloud-web provides an interactive front-end for [nodecloud](https://github.com/cloudlibz/nodecloud). NodeCloud is a standard library to get a single API on the open cloud with multiple providers.
+Making open cloud easily accessible and managed.
+
+![screenshot](screenshots/screenrecord.gif)
+
+# clone or download
```
Comment (rajikaimal) [inline README.md]: Let's use header level 2 i.e ## for sub headings like this.

Comment (amrita019) [reply README.md]: @rajikaimal I have made the requested changes, please check.

**Comment by rajikaimal** [comment]
@amrita019 Can you remove `.DS_Store` file

**Comment by rajikaimal** [comment]
@amrita019 Still there are few conflicts

**Pull request closed without merging**